### ISSUE TITLE
Avoid property sanitization

### ISSUE THREAD

**Issue by Blakeinstein** [blambright/qstylizer#2]
A perfectly valid property, 
```css
QTabBar{
	qproperty-drawBase: 0;
}
```
is changed to, 
```css
QTabBar{
	qproperty-drawbase: 0;
}
```

A more precise example would be the alignment property, 
```css
	qproperty-defaultAlignment: AlignHCenter;
```
As a side note, camel case class names also get sanitized to snake case with hyphens.
ie customTabBar -> custom-tab-bar.

**Comment by blambright** [comment]
Thanks for the heads up! Will look into fixing this early next week

**Comment by eyllanesc** [comment]
@Blakeinstein  I have already solved it, in a few moments I will send you a PR

**Comment by blambright** [comment]
Released in 0.1.8
